drivers/char/mem.c

```diff
@@ -797,8 +797,14 @@ static const struct memdev {
 #endif
 	 [5] = { "zero", 0666, &zero_fops, 0 },
 	 [7] = { "full", 0666, &full_fops, 0 },
+#if CONFIG_BSD_RANDOM
 	 [8] = { "random", 0666, &random_fops, 0 },
 	 [9] = { "urandom", 0666, &urandom_fops, 0 },
+#else
+	 [8] = { "bsd_urandom", 0666, &bsd_urandom_fops, 0 },
+	 [9] = { "bsd_urandom", 0666, &bsd_urandom_fops, 0 },
+#endif
```

Comment:
So you think the names are significant, and not just "decorative", or they are decorative but changing them is ~backwards incompatible.